contrib/kind.sh

```diff
@@ -1048,7 +1051,7 @@ install_metallb() {
   pip install -r dev-env/requirements.txt
   # Override GOBIN until https://github.com/metallb/metallb/issues/2218 is fixed.
   GOBIN="" inv dev-env -n ovn -b frr -p bgp
```

Comment:
@kyrtapz i guess running `inv dev-env -n ovn -b frr -p bgp -i ipv6` would also get kube and frr containers configured with ipv6 networks.

Reply:
Thanks, done